metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Installs and configures dnsmasq'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '0.2.1'
+version '0.2.2'
```

Comment:
You shouldn't increment the version in a pull request. The maintainer will handle that as part of the next release.